tutorials/dataframe/tdf013_InspectAnalysis.C

```diff
@@ -0,0 +1,121 @@
+/// \file
+/// \ingroup tutorial_tdataframe
+/// \notebook -nodraw
+/// Showcase registration of callback functions that act on partial results while
+/// the event-loop is running using `OnPartialResult` and `OnPartialResultSlot`.
+///
+/// \macro_code
+///
+/// \date September 2017
+/// \author Enrico Guiraud
+
+using namespace ROOT::Experimental; // TDataFrame lives in here
+
+void tdf013_InspectAnalysis()
+{
+#ifdef R__USE_IMT
```

Comment:
Can we avoid this ifdef step? The tutorial is meant to be the simplest form of illustrating a functionality. By default ROOT is imt enabled....

Reply:
I don't know any other way to have this tutorial work both on imt and noimt builds.
I could remove `EnableImplicitMT` altogether but the fact that it works seamlessly with and without imt is actually a cool part of the tutorial I think